src/vuex/store.js

```diff
@@ -0,0 +1,21 @@
+import { createStore } from 'vuex';
+
+export const store = createStore({
+    state () {
+        return {
+            autoConnect: true,
+            userSOLBalance: 0
+        }
+    },
+    mutations: {
+        setAutoConnect (state) {
+            state.autoConnect = !state.autoConnect
+        },
+        setBalance (state, payload) {
+            state.userSOLBalance = payload.balance
+        },
+        updateBalance (state, payload) {
```

Comment:
The naming here is confusing. `set` and `update` are too close together. `updateBalance` is actually incrementing the balance by a given offset so I think `incrementBalance` is more appropriate.

Even better I wouldn't include that method because it doesn't seem like it's been used in the code. Well, it is being used but wrongly (see other comment).